df_chat/drf/filters.py

```diff
@@ -0,0 +1,29 @@
+import django_filters.rest_framework
+
+from ..models import Message
+
+class MessageFilter(django_filters.rest_framework.FilterSet):
+    """
+    Filter-set class for `Message` list API.
+    All the filters that will be accessible by the filter backend
+        are defined here.
+    """
+    # For `datetime` range filter on the model's `created` field.
+    created_gte = django_filters.rest_framework.DateTimeFilter(
```

Comment:
Please rewrite imports as it suggested in [docs](https://django-filter.readthedocs.io/en/stable/guide/rest_framework.html)  

```
from django_filters import rest_framework as filters
```